core/src/main/java/org/apache/brooklyn/util/core/logbook/file/FileLogStore.java

```diff
@@ -69,23 +70,28 @@
     public final static ConfigKey<String> LOGBOOK_LOG_STORE_DATEFORMAT = ConfigKeys.newStringConfigKey(
             BASE_NAME_FILE_LOG_STORE + ".dateFormat", "Date format", "yyyy-MM-dd'T'HH:mm:ss,SSS");
 
+    public final static TimeZone UTC_TIMEZONE = TimeZone.getTimeZone("UTC");
+
     private final String filePath;
     private final Path path;
     private final String logLinePattern;
-    private final String logDateFormat;
+    private final DateFormat dateFormat;
+
 
     @VisibleForTesting
     public FileLogStore() {
         this.path = null;
         this.filePath = "";
         this.logLinePattern = LOGBOOK_LOG_STORE_REGEX.getDefaultValue();
-        this.logDateFormat = LOGBOOK_LOG_STORE_DATEFORMAT.getDefaultValue();
+        dateFormat = new SimpleDateFormat(LOGBOOK_LOG_STORE_DATEFORMAT.getDefaultValue());
```

Comment:
I'd add `this.` to dateFormat to keep it consistent

Reply:
personally i don't like the `this.` (except where it is needed)